src/lib/indexer/index.ts

```diff
@@ -220,7 +220,7 @@ export default class Indexer {
         for (const [, certificate] of changeSet.certificates) {
           switch (certificate.type) {
             case 'insert': {
-              const { type, id, ...record } = certificate
+              const { type, ...record } = certificate
```

Comment:
this was a bug which prevented us linking a `certificate_event` to a `certificate`